src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
-public class MainApp {
+import core.basesyntax.machines.Bulldozer;
+import core.basesyntax.machines.Excavator;
+import core.basesyntax.machines.Machine;
+import core.basesyntax.machines.Truck;
 
+public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machineArray = new Machine[]{ new Truck(),new Bulldozer(),new Excavator()};
```

Comment:
Kind of overloaded expression, can you please create machine instances in separate variables